Update virtual-machines-common-sizes-memory.md

steveesp opened the pull request:
Remove bandwidth ranges and labels to replace them with specific bandwidth numbers. Also updated some improved max throughput for M128 and D15v2

azurecla commented:
Hi __@beckylino__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

beckylino commented:
@JoshuaWeber could you take a look at this? Thanks!

steveesp committed:
Update virtual-machines-common-sizes-memory.md

Remove bandwidth ranges and labels to replace them with specific bandwidth numbers.
